src/pages/management/changelog/index.mdx

```diff
@@ -46,6 +46,7 @@ The developer preview version provides a preview of the upcoming changes to the
 ### Bots
 
 - Added support for the `supervisor` priority for the `groups[].priority` parameter. This applies to all **v3.x** versions of *the Configuration API*.
+- 🛠️ (2022-04-04): Increased possible size of `groups[]` parameter when adding or updating a bot.
```

Comment:
possible -> max, methinks

Reply:
I will update when @ale-kacperczyk reviews this, especially when you guys comment about the date and format of the bugfix.